gradle/test.gradle

```diff
@@ -117,6 +117,14 @@ subprojects {
     }
   }
 
+  // Facets does not extend the new runtimeOnly configurations
+  configurations.integrationTestRuntimeOnly.extendsFrom(configurations.runtimeOnly)
```

Comment:
Can you do a PR to fix facets?

Reply:
You mean a PR to fix the facets plugin at the source? Looking at that the issue has been open for a year and many PRs stay stale for months. A PR to facets would have to be paired with this workaround.

Reply:
It would be nice to improve the upstream plugins and keep workarounds out of our builds.  Oh well.